decoders/connector/dragino/wl03alb/v1.0.0/payload.test.ts

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable unicorn/numeric-separators-style */
+import { describe, expect, test } from "vitest";
+
+import { decoderRun } from "../../../../../src/functions/decoder-run";
+
+const file_path = "decoders/connector/dragino/wl03alb/v1.0.0/payload.ts" as const;
+
+function preparePayload(payloadHex: string, port: number) {
+  let payload = [
+    { variable: "payload", value: payloadHex },
+    { variable: "fport", value: port },
+  ];
+  payload = decoderRun(file_path, { payload });
+
+  const datalog = payload.find((item) => item.variable === "datalog");
+  const tdc = payload.find((item) => item.variable === "tdc");
+  const disalarm = payload.find((item) => item.variable === "disalarm");
+  const keep_status = payload.find((item) => item.variable === "keep_status");
+  const keep_time = payload.find((item) => item.variable === "keep_time");
+  const trigger_mode = payload.find((item) => item.variable === "trigger_mode");
+  const sensor_model = payload.find((item) => item.variable === "sensor_model");
+  const firmware_version = payload.find((item) => item.variable === "firmware_version");
+  const frequency_band = payload.find((item) => item.variable === "frequency_band");
+  const sub_band = payload.find((item) => item.variable === "sub_band");
+  const bat = payload.find((item) => item.variable === "bat");
+  const workmod = payload.find((item) => item.variable === "workmod");
+  const cmod = payload.find((item) => item.variable === "cmod");
+  const alarm = payload.find((item) => item.variable === "alarm");
+  const water_leak_status = payload.find((item) => item.variable === "water_leak_status");
+  const last_water_leak_duration = payload.find((item) => item.variable === "last_water_leak_duration");
+  const water_leak_times = payload.find((item) => item.variable === "water_leak_times");
+  const leak_alarm_time = payload.find((item) => item.variable === "leak_alarm_time");
+  const parse_error = payload.find((item) => item.variable === "parse_error");
+  return {
+    payload,
+    // datalog,
+    tdc,
+    disalarm,
+    keep_status,
+    keep_time,
+    sensor_model,
+    firmware_version,
+    frequency_band,
+    sub_band,
+    bat,
+    workmod,
+    cmod,
+    alarm,
+    water_leak_status,
+    last_water_leak_duration,
+    water_leak_times,
+    leak_alarm_time,
+    parse_error,
+  };
+}
+
+// time can be tested
+// describe("Port 3 unit tests", () => {
```

Comment:
Why leave this unit test commented? If it's not going to be used, it's better to remove